nautobot/core/api/urls.py

```diff
@@ -60,7 +60,8 @@
     path("swagger.yaml", SpectacularYAMLAPIView.as_view(), name="schema_yaml"),
     # GraphQL
     path("graphql/", GraphQLDRFAPIView.as_view(), name="graphql-api"),
-    # Plugins
+    # Apps
+    path("apps/", include((apps_api_patterns, "apps-api"))),
```

Comment:
Just to confirm my understanding, would this cause duplicate endpoints?

* `/api/apps/my-app/mymodel/`
* `/api/plugins/my-app/mymodel/`

Reply:
Not currently no, `/api/apps/` only contains `/installed-apps/` at present. We have a longer-term deliverable to do something with `/plugins/app/model/` and `/api/plugins/app/model` as a generalized solution.

Reply:
Thank you for clarifying, was concerned on potential bloat of the openAPI schema generation if all app api endpoints where duplicated.